libnetdata/url/url.c

```diff
@@ -208,7 +208,7 @@ char *url_decode_r(char *to, char *url, size_t size) {
 
     *d = '\0';
 
-    if(unlikely( utf8_check(to) )) //NULL means sucess here
+    if(unlikely( utf8_check((unsigned char *)to) )) //NULL means sucess here
```

Comment:
> minor - code readability HTTP CODES as defines 
> No behavior change. Just code readability

Why do not add it as a separate PR? Or change the title/OP.

Reply:
did it on request of @thiagoftsm

Reply:
I know, but now the PR changes dont reflect title/OP.

Reply:
OK, so do you want me to kick that commit out and make it separate PR *(cleaner, or let it be part of Url parser refactoring #6247)* or just change a title here

Reply:
I think better to split not related things. It feels natural for me. However title/op should always reflect changes.

This was a comment, so just keep this in mind :+1: